README.md

```diff
@@ -25,6 +25,10 @@ Once the bindings are installed you can use them with the appropriate import sta
 import "github.com/go-gl/gl/v3.3-core/gl"
 
 func main() {
+  window.MakeContextCurrent()
```

Comment:
Maybe add a `window := ... // Open a window.` comment on top? Or people will think window is some global variable.